portal-api/src/v1/controllers/utilisation-report-service/utilisation-report-upload.controller.js

```diff
@@ -108,12 +108,6 @@ const uploadReportAndSendNotification = async (req, res) => {
       return res.status(400).send();
     }
 
-    // If a report for this month/year/bank combo already exists we should not overwrite it
-    const existingReports = await api.getUtilisationReports(parsedUser?.bank?.id, parsedReportPeriod);
-    if (existingReports.length > 0) {
-      return res.status(409).send('Report already exists');
-    }
```

Comment:
It seems like the check should be made at the `portal` level rather than the `portal-api` level, but at this point I don't mind the redundancy of having another level of checking here as well so I've added it back (with updated logic)